src/clinical-trial/gateways/file-repository/clinical-trial-factory.test.ts

```diff
@@ -22,7 +22,7 @@ describe('clinical trial factory', () => {
   it('should have a clinical trial', () => {
     // GIVEN
     jest.spyOn(Date, 'now').mockReturnValue(1643566484898)
-    const clinicalTrialModel = new ClinicalTrialModel(
+    const clinicalTrialModel: ClinicalTrialModel = new ClinicalTrialModel(
```

Comment:
```suggestion
    const clinicalTrialModel = new ClinicalTrialModel(
```
Pas utile dans ce cas précis car c'est inféré avec le new ClinicalTrialModel.